config/src/main/java/org/springframework/security/config/annotation/web/AbstractRequestMatcherRegistry.java

```diff
@@ -83,7 +87,7 @@ public C anyRequest() {
 	 * @return the object that is chained after creating the {@link RequestMatcher}
 	 */
 	public C antMatchers(HttpMethod method) {
-		return antMatchers(method, new String[] { "/**" });
```

Comment:
Please move any changes that are not directly related to this issue to a different pull request or a different commit within this pull request.

Reply:
Okay -- I'll split this (and the change from a private static `valueOf` method to `HttpMethod::resolve`) into a separate commit